features/shop/cart_with_multiple_tax_rates.feature

```diff
@@ -0,0 +1,31 @@
+@ui-cart
+Feature: Apply correct tax for items with different tax rates
```

Comment:
I was thinking about it, but ``apply_correct_tax_for_items_with_different_tax_rates.feature`` is pretty long filename :smile: We should definitely think about filename <-> feature name convention, as in previously implemented scenarios we have this relation kept (e.g. ``cart_without_taxes.feature`` <-> ``Feature: Cart without taxes``).
The question is, shall we keep this convention even if filename would be long as boa snake. @michalmarcinkowski WDYT?
And if yes, shall previous scenarios be changed, to be more consistent with feature naming (``Feature: Apply correct tax for items with the same tax rate``)?